plugin/dev/test-env.sh

```
@@ -0,0 +1,178 @@
+fi
+
+# Determine terminal multiplexer
+MULTIPLEXER=""
+if _have cmux; then
+  MULTIPLEXER="cmux"
+elif _have tmux; then
+  MULTIPLEXER="tmux"
+fi
+
+# Pane commands
+PANE1_CMD="cd '$TEST_REPO' && exec \$SHELL"
```

Shellcheck (0.11.0)

[warning] PANE1_CMD appears unused. Verify use (or export if used externally).

(SC2034)